### PR TITLE
Add codeblocks for debian repo

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Related to https://github.com/jamulussoftware/jamuluswebsite/pull/938#discussion_r1278308306 copy and paste should be easy for installing the repo:

codeblocks screenshot

**Context: Fixes an issue? Related issues**
Related to comment by @mcfnord https://github.com/jamulussoftware/jamuluswebsite/pull/938#discussion_r1278308306

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
/

**Does this need translation?**
YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [ ] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/Installation-for-Linux.md [hunk 1]
```diff
@@ -25,9 +25,18 @@ We provide a repository for Debian and Ubuntu based distributions that allows au
 
 1. Open your Terminal (CTRL+ALT+T should work)
 1. *Ubuntu only* - Enable the Ubuntu "universe" repository (you can use the [GUI-based approach](https://askubuntu.com/a/148645) or [CLI-based approach](https://askubuntu.com/a/227788)).
-1. Download the repository setup script: `curl https://raw.githubusercontent.com/jamulussoftware/jamulus/main/linux/setup_repo.sh > setup_repo.sh`
-1. Make the script executable: `chmod +x setup_repo.sh`
-1. Run the setup script and install Jamulus: `sudo ./setup_repo.sh && sudo apt install jamulus`
+1. Download the repository setup script:\\
```
Comment (mcfnord) [inline wiki/en/Installation-for-Linux.md:28]: I'm unfamiliar with this slash-slash convention. What's this?

**Comment by mcfnord** [comment]
I was totally gonna do this one. Way better.

File: wiki/en/Installation-for-Linux.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Installation-for-Linux.md:28]: Basically a line break without starting a new paragraph.

Comment (mcfnord) [reply wiki/en/Installation-for-Linux.md:28]: There's a convention in Markdown of two spaces at the end of a line, which may be same, but is basically invisible so if this is a visible version of that, I'm for it.

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
I believe the weblate PR should be merged before anything else.

**Pull request merged**